test/CMakeLists.txt

```diff
@@ -44,6 +44,45 @@ option( MIOPEN_TEST_MLIR "Add tests for MLIR -- EXPERIMENTAL" ${MIOPEN_USE_MLIR}
 option( WORKAROUND_ISSUE_898 "" ON)
 option( WORKAROUND_ISSUE_936 "" ON)
 
+# Detect GPU type for testing
+if(NOT (MIOPEN_TEST_VEGA OR MIOPEN_TEST_GFX908))
+    find_program(ROCMINFO
+        NAMES rocminfo
+        PATHS
+            ${ROCM_ROOT}
+            /opt/rocm
```

Comment:
These additional paths(even for /opt/rocm) is not needed because we already append these paths to the `CMAKE_PREFIX_PATH`).

Reply:
It doesn't work on my machine. Does it work for you?

Reply:
I think that is because you are using NO_DEFAULT_PATH.

Reply:
Yeah, thanks. Made similar to ./CMakeLists.txt